### DISCUSSION TITLE
2.1.0-21000 Release

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#324] [Announcements]
### New Features

  - Gallery: Added the ability to use Albums as the start screen.
  - Settings: Added a new UI for Donations.
  - Gallery: Introduced an initial version of the Editor.
  - Editor: Improved the crop layout and added a Filters and More section.
  - Gallery: Added support for JPEG-XL images.
  - Gallery: Added the ability to set the launch screen in Settings.

### Improvements

  - AlbumSizeScreen: Slightly improved the UI.
  - Gallery: Preloaded timeline and albums viewmodels outside navigation for better performance.
  - MediaView: Dismissed trash sheet on action complete for better user experience.
  - Misc: Updated 'favorite' string for better clarity.
  - Gallery: Album thumbnails are now loaded from URI for better performance.
  - Media: Improved the Copy Sheet.
  - Gallery: Fixed Sticky Header.
  - SelectionSheet: Fixed layout on big screens for better user experience.
  - Gallery: Improved overall speed and reduced media parsing calls for better performance.
  - MediaView: Made file renaming more obvious for better user experience.

### Bug Fixes

  - Gallery: Fixed MediaStore Observer.
  - Gallery: Fixed miscellaneous issues.
  - Gallery: Moved Media Manager request to Setup to fix potential issues.
  - Gallery: Fixed various bugs and migrated to Coil for better performance and stability.

**Full Changelog**: https://github.com/IacobIonut01/Gallery/releases/tag/2.1.0-21000

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.1.0-21000'>2.1.0-21000 Release</a>.</em>

**Comment by HYDENKO** [comment]
Good time of day.
I would like to ask in connection with a significant increase in the size of the final APK file: 
- Are there any plans to develop an additional version of this application that will not have the corresponding features/components that increased the size of the APK file?

Such an additional version would be useful for Android Go devices, I assume

**Reply by MillionsToOne** [reply]
It can be optimized for smaller size but it isn't worth it in my opinion. Most android devices come with over 128GB of storage, even the lower end.
